src/block/accordion/index.php

```diff
@@ -66,3 +66,64 @@ function stackable_load_accordion_frontend_polyfill_script() {
 	}
 	add_action( 'stackable/accordion/enqueue_scripts', 'stackable_load_accordion_frontend_polyfill_script' );
 }
+
+
+if ( ! function_exists( 'stackable_render_block_accordion_FAQ_schema' ) ) {
+	function get_answer( $block, $answer ) {
+		$count_inner_blocks = count( $block[ 'innerBlocks' ] );
+		if ( $count_inner_blocks == 0 ) {
+			// check if it contains a text
+			if ( trim( strip_tags( $block[ 'innerHTML' ] ) ) != '' ) {
+				// return the text with tags
+				return $answer . trim( strip_tags( $block[ 'innerHTML'], [
+					"<h1>", "<h2>", "<h3>", "<h4>", "<h5>", "<h6>",
+					"<p>", "<a>", "<span>", "<strong>", "<em>", "<s>",
+					"<kbd>", "<code>", "<sup>", "<sub>"] ) );
```

Comment:
Would you be able to use `wp_strip_all_tags` here? I'm thinking you want to strip (for example) this:

`Hello <strong>World</strong>` to `Hello World`

```suggestion
				return $answer . wp_strip_all_tags( $block[ 'innerHTML'], true );
```

Maybe you can also use this in the `if` statement above